moj_analytics/auth0_client.py

```diff
@@ -84,6 +84,16 @@ def create(self, resource):
 
         return resource.__class__(self, response)
 
+    def update(self, resource):
+        endpoint = '{}s'.format(resource.__class__.__name__.lower())
+
+        response = self.request('PATCH', endpoint, json=resource)
+
+        if 'error' in response:
+            raise UpdateResourceError(response.message)
```

Comment:
I didn't fire an editor so apologies if it's somewhere obvious: Where is `UpdateResourceError` imported/defined?

Also, what is the `if 'error' in response` expression does? I see we're using `requests`, isn't that raising an exception? Or it doesn't raise exceptions on `4xx`/`5xx`?

Reply:
Good catch, I forgot to implement it.

Reply:
The `response` is a JSON object, which may contain an `error` field. The error response does not raise an exception in Requests.